### ISSUE TITLE
Login source identification

### ISSUE THREAD

**Issue by dissidente** [marado/TalkerNode#100]
It would be useful to be able to identify login source for logging and action purposes (ie. if a user connects through a given method or source that doesn't support color, format output without color tagging; if user connecting through Telegram bot, change some formatting to be more friendly and properly formatted, etc).

Probably the easiest way to accomplish this is to have the different methods/sources use different tcp ports and have an enum to differentiate them (telnet, telnet-ssl, web, Telegram bot, etc)

Listening port(s) would be configured as an option in talker.db.

**Comment by marado** [comment]
TBH, I am a firm believer that the work you propose (colorless port, for eg.) should be the work of the client, not the server. i.e., if an (intermediate, sure) client like the telegram client does not support color, it should be up to it to strip all that ansi output that might be coming from TalkerNode...

I feel, in a way, that the moment that we go down the slippery slope of having TalkerNode customizations to allow for an ssl client, or a telegram bot client, or a web client... well, then it would possibly make more sense that TalkerNode itself should provide that telnet-ssl port, that telegram bot, or that web interface directly.

Still, I do not oppose that we have all the plumbing necessary to, at least, have more than one connection port, if the talker is configured as such, and have that port number be saved in a number of places (eg, the auth_log telling which port is a connection to, maybe the user database reflect that, or even have some info about that visible with some commands, like .exa, .last or .who, .people if we end up doing that one...).

tldr: I do not plan to work on this (at least in the foreseeable future), but I will probably accept PRs for it.
